config/v1/0000_10_config-operator_01_build.crd.yaml

```diff
@@ -216,6 +216,12 @@ spec:
                                 description: Specify whether the Secret or its key
                                   must be defined
                                 type: boolean
+                forcePull:
+                  description: ForcePull overrides, if set, the equivalent value in
+                    the builds, i.e. false disables force pull for all builds, true
+                    enables force pull for all builds, independently what each build
```

Comment:
```suggestion
                    enables force pull for all builds, independently of what each build
```

Reply:
but i thought we were dropping the default field.

Reply:
weird, i was sure i had removed that, trying again ...

Reply:
fixed in current push